src/apphosting/index.ts

```diff
@@ -121,6 +148,7 @@ export async function doSetup(
   });
 
   await setDefaultTrafficPolicy(projectId, location, backendId, branch);
+  logSuccess(`Successfully created backend:\n\t${backend.name}`);
```

Comment:
Remove, this is already on L138

Reply:
ugh. merge issues. I had previously moved and now it got turned into an add. TY for the catch.